src/components/BreedPhotos/BreedPhotos.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import ReactPaginate from 'react-paginate';
+import ImageCard from '../ImageCard/ImageCard';
+import formatBreedName from '../../helpers/formatName';
+import { BreedPhotosGrid, StyledMain, StyledPaginateContainer } from './styles';
+
+type BreedPhotosProps = {
+    breedName: string;
+    itemsPerPage: number;
+};
+
+type ItemProps = {
+    currentItems: Array<string>;
+}
+
+function Items({ currentItems }: ItemProps) {
+    return (
+      <>
+        {currentItems &&
+          currentItems.map((item) => (
+            <ImageCard key={item} imageUrl={item} />
+          ))}
+      </>
+    );
+  }
+
+export default function BreedPhotos({ breedName, itemsPerPage }: BreedPhotosProps) {
+    // Set empty list of items.
+    const [items, setItems] = useState<string[]>([]);
+    const [currentItems, setCurrentItems] = useState<string[]>([]);
+    const [pageCount, setPageCount] = useState<number>(0);
+    // Set page offsets.
+    const [itemOffset, setItemOffset] = useState<number>(0);
+
+    const getBreedPhotos = useCallback(() => {
+        if(breedName.includes('-')) {
+            breedName = breedName.replace('-', '/');
```

Comment:
I believe it's somewhat dangerous to reassign a component prop? WDYT?

Reply:
You are totally right. I just checked React documentation and this is a bad practice. I should have created another variable to store the modified breedName.